scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -85,6 +87,8 @@
 
     private static final String STR_JOB = " jobs";
 
+    private static final String HEADER_PA_ERROR = "proactive_error";
```

Comment:
`proactive_error` looks a bit weird to me. Generally, we refer to the backend of ProActive as 'the server'..

Reply:
Nope, In addition, it might avoid any "kind of" future conflicts if someone decides to add a header and call it `error`